### ISSUE TITLE
[BUG] Alert should be able to fit 320 px width

### ISSUE THREAD

**Issue by alxzak** [kirbydesign/designsystem#4018]
NET–AMP315 – Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality

What is the issue?
Actual result
[Issue]
The "alert" dialog does not appear when the page has a viewport width of 320 CSS pixels. 

[User Impact]
Users with low vision who need to resize text will be prevented from accessing this content.

[Code Reference]
N/A

[Steps to Reproduce]
1. Open Chrome DevTools.
2. Activate "Toggle device toolbar" (laptop and cell phone icon).
3. In "Dimensions", select "Responsive".
4. In "Width", enter "320".
5. Refresh the page.
6. Notice that content disappears, is cut off, or overlaps.

Steps to reproduce
1. Open Chrome DevTools.
2. Set the dev tools to doc on the side.
3. Resize the dev tools so the page viewport is 320CSS pixels
5. Refresh the page.
6. Notice that content disappears, is cut off, or overlaps.

How to resolve the issue
Recommendation
[Recommendation]
Ensure pages reflow without requiring two-dimensional scrolling without loss of content or functionality. When the page has a viewport width of 320 CSS pixels, all content and functionality must still be available without requiring horizontal scrolling.

Moving content to an accessible show/hide control, such as a hamburger menu, is acceptable.

Content that requires two dimensions to be understood, such as a map or data table, is exempt from this requirement.

**Comment by alxzak** [comment]
<img width="1918" height="968" alt="Image" src="https://github.com/user-attachments/assets/afabcc95-3575-4fa5-9e09-39ea518c2a4a" />

**Comment by alxzak** [comment]
<img width="1918" height="956" alt="Image" src="https://github.com/user-attachments/assets/1a013328-c787-441b-9f63-c174f161b89c" />

**Comment by alxzak** [comment]
Should be corrected by #3578

**Comment by RasmusKjeldgaard** [comment]
Linked issue is fixed, and I can no longer reproduce in cookbook, so I am closing this one.
